src/displayable.d.ts

```diff
@@ -0,0 +1,3 @@
+import type { Color } from './common';
+
```

Comment:
I noticed that when in the file, for example in `displayable.js`, when there is an import which looks like `import converter from './converter.js';` because of `.js` in the path, at least in WebStorm not showing the type hint and autocomplition is not working.